src/main/kotlin/no/nav/familie/ef/søknad/api/DokumentasjonsbehovController.kt

```diff
@@ -28,4 +31,16 @@ class DokumentasjonsbehovController(private val søknadClient: SøknadClient) {
 
         return ResponseEntity.ok(dokumentasjonsbehovDto)
     }
+    @PostMapping("/person")
+    fun hentDokumentasjonsbehovForPerson(@RequestBody personIdent: PersonIdent): ResponseEntity<List<DokumentasjonsbehovDto>> {
+        if (!personIdentErLikInnloggetBruker(personIdent.ident)) {
+            throw ApiFeil("Fnr fra token matcher ikke fnr på søknaden", HttpStatus.FORBIDDEN)
+        }
+        val søknader = søknadClient.hentSøknaderForPerson(personIdent.ident)
+
+        val dokumentasjonsbehov = søknader.map { søknad -> søknadClient.hentDokumentasjonsbehovForSøknad(UUID.fromString(søknad)) }
+
```

Comment:
Burde vi ha ett api i mottak som henter dokumentasjonsbehovet for en gitt personident, sånn att man ikke trenger å gjøre dette i 2 kall?

Reply:
Enig!